tests/template_tests/syntax_tests/test_basic.py

```diff
@@ -164,6 +164,88 @@ def test_basic_syntax19(self):
         else:
             self.assertEqual(output, "")
 
+    @setup({"basic-syntax19a": "{{ klass.in_template }}"})
```

Comment:
```suggestion
    @setup({"basic-syntax19a": "{{ var.in_template }}"})
```
(this is for a few places) I think other tests are using `var` and I think `klass` might be german

Reply:
Happy to do the change. `klass` or `cls` sometimes are used in Python to avoid using the `class` keyword. No need for that here.

Reply:
Ah today I learned :grin: